PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -322,9 +322,19 @@ def miniAOD_customizeCommon(process):
 
 
 def miniAOD_customizeMC(process):
+    task = getPatAlgosToolsTask(process)
+    #GenJetFlavourInfos
+    process.load("PhysicsTools.JetMCAlgos.HadronAndPartonSelector_cfi")
+    process.selectedHadronsAndPartons.particles = "genParticles"
```

Comment:
it's already genParticles in PhysicsTools/JetMCAlgos/python/HadronAndPartonSelector_cfi.py
please do not override.
If a change is needed, make a clone and modify it.